ui/src/app/shared/workflow/wnode/pipeline/wnode.pipeline.component.ts

```diff
@@ -36,7 +36,7 @@ export class WorkflowWNodePipelineComponent {
     displayLogs() {
         if (this.noderun) {
             this._router.navigate(['node', this.noderun.id], {
-                relativeTo: this._activatedRoute,
+                relativeTo: this._activatedRoute, queryParams: { name: this.node.name }
```

Comment:
Looks like this data should be retreive directly from the noderun, what will happen is I set manually this data in the url with random data?

Reply:
This is just to have a nice title in browser tabs.
The process that set the title is located in app.component.ts and just analyze the url.

Reply:
Mokay